chain/near-jsonrpc-traits/src/handler.rs

```diff
@@ -0,0 +1,155 @@
+use std::future::Future;
+
+pub trait JsonRpcHandlerExt {
+    fn send_tx(
```

Comment:
Can you consider moving the docstrings from the `near-jsonrpc` library, at least if they were there, to this new crate? What do you think?

Reply:
I move them, but only `validators_ordered` and `maintenance_windows` had docs